mailgun-relayery/timestamp-db/database.go

```diff
@@ -0,0 +1,371 @@
+package timestamp_db
```

Comment:
merge with the other database package, lump it all in one file so that transaction code is in the same file.